### ISSUE TITLE
Following tutorial in RStudio IDE is hanging at preview

### ISSUE THREAD

**Issue by cderv** [quarto-ext/manuscript-template-rstudio#2]
From https://community.rstudio.com/t/quarto-manuscripts-trouble-rendering-preview/179051, following https://quarto.org/docs/manuscripts/authoring/rstudio.html

There is something causing the initial rendering to hang - no error message.

**Comment by cderv** [comment]
Possibly a RStudio IDE related issue as using Command line in terminal works as expected 

```
quarto preview index.qmd --to html --no-watch-inputs --no-browse
```

In background job I get 
````
==> quarto preview index.qmd --to html --no-watch-inputs --no-browse

processing file: index.qmd
                                                                                                            
output file: index.knit.md

Rendering HTML preview [index.qmd]
pandoc 
  to: html
  output-file: >-
    C:\Users\chris\Documents\DEV_OTHER\quarto-test-manuscript\manuscript-tutorial\index.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png
  toc: true
  number-sections: true
  
metadata
  
  document-css: false
  link-citations: true
  date-format: long
  lang: en
  manuscript:
    article: index.qmd
    notebooks:
      - notebook: notebooks/explore-earthquakes.qmd
      - notebook: index.qmd
        title: Article Notebook
    mecaFile: index-meca.zip
  notebook-preview-options:
    back: true
  theme: cosmo
  title-block-style: manuscript
  lightbox: auto
  comments:
    hypothesis: true
  title: La Palma Earthquakes
  author:
    - name: Steve Purves
      orcid: 0000-0002-0760-5497
      corresponding: true
      email: steve@curvenote.com
      roles:
        - Investigation
        - Project administration
        - Software
        - Visualization
      affiliations:
        - Curvenote
    - name: Rowan Cockett
      orcid: 0000-0002-7859-8394
      corresponding: false
      roles: []
      affiliations:
        - Curvenote
  keywords:
    - La Palma
    - Earthquakes
  abstract: |
    In September 2021, a significant jump in seismic activity on the island of La Palma (Canary Islands, Spain) signaled the start of a volcanic crisis that still continues at the time of writing. Earthquake data is continually collected and published by the Instituto Geográphico Nacional (IGN). ...
  plain-language-summary: |
    Earthquake data for the island of La Palma from the September 2021 eruption is found ...
  key-points:
    - A web scraping script was developed to pull data from the Instituto Geogràphico Nacional into a machine-readable form for analysis
    - Earthquake events on La Palma are consistent with the presence of both mantle and crustal reservoirs.
  date: last-modified
  bibliography:
    - references.bib
  citation:
    container-title: Earth and Space Science
  google-scholar: true
  toc-location: left
  clear-hidden-classes: none
  remove-hidden: all
  unroll-markdown-cells: true
  
Output created: index.html

Terminating existing preview server....DONE

Preparing to preview
````

In Terminal I get:
````
 quarto preview index.qmd --to html --no-watch-inputs --no-browse
++ Activating rlang global_entrace



processing file: index.qmd

output file: index.knit.md

Rendering HTML preview [index.qmd]
pandoc
  to: html
  output-file: >-
    C:\Users\chris\Documents\DEV_OTHER\quarto-test-manuscript\manuscript-tutorial\index.html
  standalone: true
  section-divs: true
  html-math-method: mathjax
  wrap: none
  default-image-extension: png
  toc: true
  number-sections: true

metadata

  document-css: false
  link-citations: true
  date-format: long
  lang: en
  manuscript:
    article: index.qmd
    notebooks:
      - notebook: notebooks/explore-earthquakes.qmd
      - notebook: index.qmd
        title: Article Notebook
    mecaFile: index-meca.zip
  notebook-preview-options:
    back: true
  theme: cosmo
  title-block-style: manuscript
  lightbox: auto
  comments:
    hypothesis: true
  title: La Palma Earthquakes
  author:
    - name: Steve Purves
      orcid: 0000-0002-0760-5497
      corresponding: true
      email: steve@curvenote.com
      roles:
        - Investigation
        - Project administration
        - Software
        - Visualization
      affiliations:
        - Curvenote
    - name: Rowan Cockett
      orcid: 0000-0002-7859-8394
      corresponding: false
      roles: []
      affiliations:
        - Curvenote
  keywords:
    - La Palma
    - Earthquakes
  abstract: |
    In September 2021, a significant jump in seismic activity on the island of La Palma (Canary Islands, Spain) signaled the start of a volcanic crisis that still continues at the time of writing. Earthquake data is continually collected and published by the Instituto Geográphico Nacional (IGN). ...
  plain-language-summary: |
    Earthquake data for the island of La Palma from the September 2021 eruption is found ...
  key-points:
    - A web scraping script was developed to pull data from the Instituto Geogràphico Nacional into a machine-readable form for analysis
    - Earthquake events on La Palma are consistent with the presence of both mantle and crustal reservoirs.
  date: last-modified
  bibliography:
    - references.bib
  citation:
    container-title: Earth and Space Science
  google-scholar: true
  toc-location: left
  clear-hidden-classes: none
  remove-hidden: all
  unroll-markdown-cells: true

Output created: index.html

Terminating existing preview server....DONE

Preparing to preview

Watching files for changes
Browse at http://localhost:5969/
````

**Comment by dragonstyle** [comment]
This should be resolved in a newer pre-release build of Quarto 1.4. See https://github.com/quarto-dev/quarto-cli/issues/8050
